### DISCUSSION TITLE
Cannot Access Prologue Docs in Safari

### DISCUSSION THREAD

**Discussion by mtanco** [h2oai/wave#1102] [General]
When in Safari, I cannot access any of the prologue doc pages like the Change Log. When I click on one of these the browser loads for a couple seconds (blue bar at the top) and when it's done nothing has changed. 

I do not see this problem in Chrome and I do not see the problem with any non-prologue pages.

**Comment by mturoci** [comment]
Works fine for me in Safari, what's your Safari version?

**Reply by mtanco** [reply]
.... it's working now... I guess I should restart my computer first next time
